notifications/notificationHandler.ts

```diff
@@ -0,0 +1,71 @@
+import * as Notifications from 'expo-notifications'
+import { Platform } from 'react-native'
+
+const askPermissions = async () => {
+  Notifications.getPermissionsAsync().then((status) => {
+    if (!status.granted) {
+      console.log('Requesting Permission')
+      Notifications.requestPermissionsAsync()
+    }
+  })
+}
+
+const deleteNotifications = async () => {
+  Notifications.cancelAllScheduledNotificationsAsync()
+}
+
+const iosScheduler = (weekday: number[], time: Date) => {
+  if (weekday !== [-1]) {
+    Notifications.cancelAllScheduledNotificationsAsync()
+    weekday.forEach((v) => {
+      if (v !== -1) {
+        Notifications.scheduleNotificationAsync({
+          content: {
+            title: 'Practice your meditation today  🙇‍♂️',
+          },
+          trigger: {
+            hour: time.getHours(),
+            minute: time.getMinutes(),
+            weekday: v,
+          },
+        })
+      }
+    })
+  }
+}
+
+const AndroidSchedular = (weekday: number[], time: Date) => {
```

Comment:
Stick with camelCase instead of mixing the two. As you already have `iosSchedular` in camelCase and the rest of the codebase is camelCase for the most part.

Reply:
done